riscos_toolbox/objects/colourdbox.py

```diff
@@ -15,21 +15,21 @@ class ColourDbox(Object):
 
     @property
     def colour(self):
-        colour = swi.swi("Toolbox_ObjectMiscOp","III;i", 0,self.id, 1)
+        colour = swi.swi("Toolbox_ObjectMiscOp", "III;i", 0, self.id, 1)
```

Comment:
Isn't method 1 => GetDialogueHandle?

From the header here:

```
/****************************************************************************
 * ColourDbox Methods                                                       *
 ****************************************************************************/


#define ColourDbox_GetWimpHandle           0
#define ColourDbox_GetDialogueHandle       1
#define ColourDbox_SetColour               2
#define ColourDbox_GetColour               3
#define ColourDbox_SetColourModel          4
#define ColourDbox_GetColourModel          5
#define ColourDbox_SetNoneAvailable        6
#define ColourDbox_GetNoneAvailable        7
```

Am I missing something subtle?

Reply:
It is, and it's also just wrong because GetColour needs a buffer to fill in with the values, rather than just returning an int.